lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -67,7 +45,7 @@ public function register_routes() {
 				array(
 					'methods'             => WP_REST_Server::READABLE,
 					'callback'            => array( $this, 'get_items' ),
-					'permission_callback' => array( $this, 'get_font_faces_permissions_check' ),
+					'permission_callback' => array( $this, 'get_items_permissions_check' ),
```

Comment:
Overriding the default permissions check methods, as requested here: https://github.com/WordPress/gutenberg/pull/57688#discussion_r1456276097